content/posts/hello-world/index.md

```diff
@@ -14,7 +14,8 @@ draft: false
 
 We'd like to announce the creation of a new non-profit organization[^1] that we call **2i2c**.
 
-[^1]: 2i2c's host organization is the [International Computer Science Institute](http://www.icsi.berkeley.edu/), a 501(c)(3) charitable organization. We thank ICSI for its collaboration and leadership in launching 2i2c!
+[^1]: **EDIT 08-2021: We are now a fiscally-sponsored project of [Code for Science and Society](https://codeforscience.org)**. 2i2c's host organization is the [International Computer Science Institute](http://www.icsi.berkeley.edu/), a 501(c)(3) charitable organization. We thank ICSI for its collaboration and leadership in launching 2i2c! 
```

Comment:
> 2i2c's host organization is

`is`? 
Is ICSI our host org even when we are fiscally sponsored by CSS?

Reply:
No - I just didn't want to alter the language because this is a blog post. So at the time it was true but not anymore which is why I added the strikeout. Is it not clear enough?

Reply:
> this is a blog post

OK, thanks for the additional context I missed in the review!